### ISSUE TITLE
BMP secretion

### ISSUE THREAD

**Issue by RLovering** [geneontology/go-annotation#4052]
Hi Pascale

Are we planning to keep BMP secretion and Child terms?
GO:0038055 BMP secretion

Or will these be production terms?

Thanks

Ruth

**Comment by pgaudet** [comment]
Thanks for binging this up. There are 5 EXP annotations by FlyBase and ZFIN. From glancing over the abstracts it seems like 'regulation of BMP signaling' would work for these genes - **@hattrill @sramachand Can you please check?** 

In any case the 'BMP' secretion term is not used. It can at least be marked DO NOT ANNOTATE. 

Thanks, Pascale

**Comment by RLovering** [comment]
But could we have a term to enable us to capture regulation of ligand production, which would be the parent to the regulation of cytokine etc production terms? Only I think the BMP secretion term should change to BMP production and there will be other ligands that we don't have specific terms, eg there are no growth factor production terms or terms for ligands such as Wnt, BMP, morphogens - just realised there are approx 600 terms with 'secretion' search eg
GO:0046879 hormone secretion
I guess this was a project that we didn't finish ;(

**Comment by pgaudet** [comment]
> I guess this was a project that we didn't finish ;(

right!!!

So, you propose we have 'regulation of production' and/or 'regulation of secretion' for each ligand? Or a generic type and use 'has input'?

**Comment by sramachand** [comment]
So, the annotation in ZFIN is to "positive regulation of BMP secretion". The annotation refers to the actual effect on the amount of secreted extracellular BMP. Is the proposal to use "regulation of ligand production"? And production encompasses the synthesis and secretion?

**Comment by pgaudet** [comment]
> production encompasses the synthesis and secretion?

yes, this is how it's defined. However I dont' think we should get rid of all regulation of secretion terms without looking more closely at whether some of these are specific pathways. What I saw from the BMP papers was that it seemed that secretion of the ligand was indeed regulated. But we could have a more general term for this (regulation of signaling ligand secretion?)

**Comment by addiehl** [comment]
If there is detailed data about the regulation of secretion itself, then it is appropriate to have a secretion term for a particular cytokine.  The production terms were intended to annotate gene products that affect the level of a cytokine where the assay itself cannot distinguish between regulation of synthesis or regulation of secretion, and there are use cases for both production terms and secretion terms.

**Comment by RLovering** [comment]
just to clarify, my interpretation of the discussions that we have had is that:

For ligands (or other entities) that are held in vesicles and then are secreted following a specific signaling pathway we would keep the secretion option, which would require evidence that there is no new protein synthesis or that, based on existing knowledge, the annotation applied referred to a known pathway that caused regulated secretion of the ligand (or other entity).
For ligands (or other entities) that require synthesis before secretion the GOC was going to provide just production terms and that only +/-/regulation of production can be used. Proteins involved in the general process of exocytosis, protein synthesis, or transcription would be associated with the more general terms rather than specific ligand terms.

I am not sure if this has been recorded in any GOC documentation. Hopefully I have remembered these discussions appropriately
